### ISSUE TITLE
Accessibility - form labels are not associated with inputs on "Sign up" page

### ISSUE THREAD

**Issue by kaitlinnewson** [IQSS/dataverse#7771]
Tested in version 5.3 on Harvard Dataverse.

On the [sign up form page](https://dataverse.harvard.edu/dataverseuser.xhtml?editMode=CREATE), it appears that the form labels are not properly associated with their input fields, which creates an accessibility issue.

For example, the "Username" label has `for="userNameEmail"`, but the text input field for the username has the ID set as `id="dataverseUserForm:accountInfoView:userName"`. This issue occurs for all of the labels on this page.

![WAVE Accessibility test of the Dataverse sign up page](https://user-images.githubusercontent.com/6903515/113912119-c88e4900-97a8-11eb-9bf4-48495dabaabb.png)

**Comment by scolapasta** [comment]
At a quick glance, it looks as if the code on `dataverseuser.xhtml` is using simple <label> tags instead of <h:outputLabel> or  <p:outputLabel>.

(and then references userNameEmail instead of just userName for the example you gave)
